src/Engine/ProtoCore/DSASM/Mirror/ExecutionMirror.cs

```diff
@@ -723,23 +710,21 @@ public Obj GetWatchValue()
             return retVal;
         }
         
-        public Obj GetValue(string name, int block = 0, int classcope = Constants.kGlobalScope)
+        /// <summary>
+        /// Return the stack value of specified symbol.
```

Comment:
It seems that this method just looks for Symbol in global scope?? If so can we update the method name as well as documentation here to reflect that?